.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
-  extends:   ["eslint-config-mitodl", "eslint-config-mitodl/jest", "plugin:testing-library/react"],
-  plugins: ["testing-library"],
+  extends:        ["eslint-config-mitodl", "eslint-config-mitodl/jest", "plugin:testing-library/react"],
+  plugins:        ["testing-library"],
+  ignorePatterns: ["**/build/**"]
```

Comment:
Previously we were only linting files under `frontends/<package>/src/` (via CLI argument).

Now we lint/format all files under `frontends/<package>`, except those in node_modules and build directories.